lib/gds_api/test_helpers/performance_platform/data_out.rb

```diff
@@ -17,6 +17,70 @@ def stub_data_set_not_available(slug)
         def stub_service_not_available
           stub_request(:any, /#{PP_DATA_OUT_ENDPOINT}\/.*/).to_return(status: 503)
         end
+
+        def stub_page_statistics(slug, group_by, collect, response_body = {})
+          params = {
+              filter_by: "pagePath:" + slug,
+              group_by: group_by,
+              collect: collect,
+              duration: 42,
+              period: "day",
+              end_at: Date.new.to_time.getutc
+          }
+
+          data_out = GdsApi::PerformancePlatform::DataOut.new(PP_DATA_OUT_ENDPOINT)
+          query_string = data_out.send(:query_string, params)
```

Comment:
Using the implementation to get the query string version of the params feels a bit weird (on first reading I thought we were actually making the API call!).  I hoped there might be a simpler way and it turns out that [Webmock can take a `query` argument](https://github.com/bblimke/webmock#matching-query-params-using-hash) that takes a hash to avoid this.

```ruby
stub_http_request(:get, "#{PP_DATA_OUT_ENDPOINT}/data/govuk_info/page-statistics")
  .with(query: params)
  .to_return(status: 200, body: response_body.to_json)
```